lib/utils/ReporterStats.js

```diff
@@ -248,6 +248,10 @@ class ReporterStats extends RunnableStats {
         // Remove the screenshot data to reduce RAM usage on the parent process
         if (type === 'screenshot') {
             runner.data = null
+        } else if (type === 'result' && runner.requestOptions.uri.path.includes('screenshot')) {
+            runner.body = null
+        } else if (type === 'aftercommand' && runner.command === 'saveDocumentScreenshot') {
```

Comment:
This is not a WebdriverIO screenshot. Any chance we can restrict this to API commands? Because in case that `wdio-screenshot` renames their API commands this problem will reoccur.